systemd/afterburn-checkin.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Descitpion=Afterburn Check In
```

Comment:
`Description`

nit: For consistency with the other units, should be e.g. `Afterburn (Check In)`.

Reply:
:+1: will update